sample-lnd.conf

```diff
@@ -1135,6 +1135,13 @@ litecoin.node=ltcd
 ; testing with embedded etcd.
 ; db.etcd.embedded_log_file=/path/etcd.log
 
+[postgres]
+; Postgres connection string.
+; db.postgres.dsn=postgres://lnd:lnd@localhost:45432/lnd?sslmode=disable
+
+; Postgres connection timeout. Leave unset for no timeout. Valid time units
```

Comment:
Should we use the same wording here as in the command line description? There it says "Set to zero to disable".